.golangci.yml

```diff
@@ -88,7 +88,7 @@ linters:
           disabled: true
         - name: line-length-limit
           arguments:
-            - 160
+            - 210
```

Comment:
If you are doing this, I would simply disable it.

The other option is to add `//nolint:revive` directive where needed